sections/model_overview/main.tex

```diff
@@ -1 +1,168 @@
-\section{Overview of game theoretic model}
\ No newline at end of file
+\section{Overview of game theoretic model}
+
+The problem studied is a 3-player game theoretic scenario between two queueing 
+systems and a service that distributes individuals to them. 
+Each player in this game has its own objectives and cares about meeting its own 
+targets.
+More specifically, the queueing systems' objective is captured by an upper bound
+of the time that a fixed proportion of individuals need to spend in the system, 
+while the distributor aims to minimise the time that its individuals 
+are blocked.  
+
+The queueing systems are designed in such a way where they can accept two types
+of individuals. 
+These are the individuals that the distributor allocates to them and 
+other individuals from other sources. 
+Each queueing system may then choose to block the individuals that arrive from 
+the distributor when the system reaches a certain capacity. 
+Thus, the set of strategies that they may choose to play in the game are defined 
+as the set of possible choices that the two queueing systems have. 
+In other words, each queueing system is given \( N_A \) and \( N_B \) number of 
+strategies where \( N_A \) and \( N_B \) are the total capacities of the two 
+queueing systems.
+The \textit{threshold} parameter is thus used to capture the chosen 
+strategy for each hospital where \( T_A \in \left[1, N_A \right] \) and 
+\( T_B \in \left[1, N_B \right] \)
+Their choice of strategy will then rely solely on satisfying their own 
+objective, which is to make sure that the waiting time of a proportion of 
+individuals will be below the predefined target time.
+
```

Comment:
```suggestion
individuals will be below the predefined target time.

This is shown diagrammatically in Figure~ref{fig:diagram_of_queueing_system}.

% TODO Draw diagram of queueing system
% This should include the parameters
% and clearly show the decision variable.
% Note: only do this for a single queueing system.

```

Reply:
By diagram do you mean a general queuing diagram (like the picture below) or the Markov chain one?

![image](https://user-images.githubusercontent.com/20097997/114890543-058ab900-9e03-11eb-8ba8-4be77e394505.png)